argo/templates/template.yaml

```diff
@@ -0,0 +1,68 @@
+{{- $version := default "latest" .Chart.AppVersion }}
+apiVersion: argoproj.io/v1alpha1
```

Comment:
For reference, here's the version in ClinVar: https://github.com/DataBiosphere/clinvar-ingest/blob/master/orchestration/templates/extract-xml-to-json.yaml

I'll comment where I changed things